### ISSUE TITLE
Connection failed (Telemed MicrUs Pro scanner)

### ISSUE THREAD

**Issue by Ginkbel** [PlusToolkit/PlusLib#1215]
Hi,

I'm trying to set up Plus Server Launcher for Telemed scanner but run into the error:

  Connection failed!
  
  Please select another device set and try again!

The log message is:

time|level|timeoffset|message|location
111424_225425.276|INFO|000.002000|> Software version: Plus-2.9.0.85881d7b - Win64| in vtkPlusLogger(52)
111424_225425.279|INFO|000.005000|  Software version: Plus-2.9.0.85881d7b - Win64| in E:\D\PTNPTe64b\PlusApp\PlusServerLauncher\PlusServerLauncherMainWindow.cxx(148)
111424_225425.279|INFO|000.006000| Logging at level 3 to file: C:/Users/PlusApp-2.9.0.20241029-Telemed-Win64/bin/PlusServerLauncher.exe/Output/111424_225425_PlusLog.txt| in E:\D\PTNPTe64b\PlusApp\PlusServerLauncher\PlusServerLauncherMainWindow.cxx(149)
111424_225426.039|INFO|000.765000| Supported devices: 
  - 3dConnexion (ver: Plus-2.9.0)
  - AuroraTracker (ver: NDICAPI-3.7)
  - BkProFocusOem (ver: Plus-2.9.0)
  - ChRobotics (ver: Plus-2.9.0)
  - Epiphan (ver: Plus-2.9.0)
  - FakeTracker (ver: Plus-2.9.0)
  - GenericSerialDevice (ver: Plus-2.9.0)
  - ICCapturing (ver: The Imaging Source UDSHL-3.5)
  - ImageProcessor (ver: Plus-2.9.0)
  - IntelRealSense (ver: Plus-2.9.0)
  - Microchip (ver: Plus-2.9.0)
  - MmfVideo (ver: Plus-2.9.0)
  - NDITracker (ver: NDICAPI-3.7)
  - NoiseVideo (ver: Plus-2.9.0)
  - OpenCVVideo (ver: Plus-2.9.0)
  - OpenIGTLinkTracker (ver: OpenIGTLink v3.1.0)
  - OpenIGTLinkVideo (ver: OpenIGTLink v3.1.0)
  - OptiTrack (ver: Plus-2.9.0)
  - OpticalMarkerTracker (ver: Plus-2.9.0)
  - PhidgetSpatial (ver: Plus-2.9.0)
  - PolarisTracker (ver: NDICAPI-3.7)
  - SavedDataSource (ver: Plus-2.9.0)
  - TelemedVideo (ver: Telemed version unknown| in E:\D\PTNPTe64b\PlusApp\PlusServerLauncher\PlusServerLauncherMainWindow.cxx(153)
111424_225426.047|INFO|000.774000| Server host name: | in E:\D\PTNPTe64b\PlusApp\PlusServerLauncher\PlusServerLauncherMainWindow.cxx(184)
111424_225426.059|INFO|000.785000| Server IP addresses: 192.168.1.234, 127.0.0.1| in E:\D\PTNPTe64b\PlusApp\PlusServerLauncher\PlusServerLauncherMainWindow.cxx(204)
111424_225426.059|INFO|000.785000| Start remote control server at port: 18904| in E:\D\PTNPTe64b\PlusApp\PlusServerLauncher\PlusServerLauncherMainWindow.cxx(213)

The device works in fCal as per the image 

![image](https://github.com/user-attachments/assets/d51878eb-a4b4-4f34-ae06-1c9aa2436d35)

**Comment by ungi** [comment]
Hi, have you tried running Plus Server Launcher in compatibility mode in Windows?

**Comment by Ginkbel** [comment]
Yup, didn't work, but I think I figured it out. It doesn't work because of the special characters in the "C:\Users\<username>" path. 

When I logged into a different account it works without a problem. Will install the app in different folder.

Also, is there a way to modify the parameters of the ultrasound scanner that is being streamed via the server? For example gain, focus, depth and other parameters?
